src/pages/Donate/DonateFormDropInPaymentWrapper.vue

```diff
@@ -181,7 +181,7 @@ export default {
 				mutation: braintreeCreateAutoDepositSubscription,
 				variables: {
 					paymentMethodNonce: nonce,
-					amount: '0.00',
+					amount: this.donateAmountAsString,
```

Comment:
This value must represent the total amount of the monthly deposit. @eddieferrer @emuvente